be/src/service/doris_main.cpp

```diff
@@ -166,8 +166,8 @@ auto instruction_fail_to_string(InstructionFail fail) {
     case InstructionFail::ARM_NEON:
         ret("ARM_NEON");
     }
-    LOG(FATAL) << "__builtin_unreachable";
```

Comment:
why not core dump? if returned, then it will core in other place.

Reply:
now,how do i resolve the issue